tests/test_data_loader.py

```diff
@@ -384,3 +386,13 @@ def test_end_of_dataloader(self):
         # Test it also works on the second iteration
         for idx, _ in enumerate(dataloader):
             self.assertEqual(dataloader.end_of_dataloader, idx == 3)
+
+    def test_end_of_dataloader_dispatcher(self):
+        accelerator = Accelerator()
```

Comment:
Couldn't make the test work without the line. `AcceleratorState` raises "Please make sure to properly initialize your accelerator via `accelerator = Accelerator()` before using any functionality from the `accelerate` library."